internal/ini/visitor.go

```diff
@@ -125,8 +164,27 @@ func (t Sections) List() []string {
 // Section contains a name and values. This represent
 // a sectioned entry in a configuration file.
 type Section struct {
-	Name   string
+	// Name is the Section profile name
+	Name string
+
+	// values are the values within parsed profile
 	values values
+
+	// Errors is the list of errors
+	Errors []error
+
+	// Logs is the list of logs
+	Logs []string
```

Comment:
This Logs field is never used as of now. The ideas was to stash logs, and only call them if related profile is referenced. But that doesn't work well for profiles that are dropped without a similar profile used in place of it.